app/schemas.py

```diff
@@ -478,7 +478,7 @@ def get_reply_to_text(self, template):
         return template.get_reply_to_text()
 
     def get_letter_attachment(self, template):
-        return template.letter_attachment_id
+        return template.letter_attachment.serialize() if template.letter_attachment else None
```

Comment:
Do we need to worry about this return value changing (ie the raw ID already in use by admin) or are we free to change this still?